src/dotnet/commands/dotnet-help/LocalizableStrings.resx

```diff
@@ -273,4 +273,19 @@
   <data name="BuildServerDefinition" xml:space="preserve">
     <value>Interact with servers started by a build.</value>
   </data>
+  <data name="DevCertsDefinition" xml:space="preserve">
+    <value>Create and manage development certificates.</value>
+  </data>
+  <data name="EfDefinition" xml:space="preserve">
+    <value>Entity Framework Core command-line tools.</value>
+  </data>
+  <data name="SqlCacheDefinition" xml:space="preserve">
+    <value>SQL Server cache command-line tools.</value>
+  </data>
+  <data name="UserSecretsDefinition" xml:space="preserve">
+    <value>Manage development user secrets.</value>
+  </data>
+  <data name="WatchDefinition" xml:space="preserve">
+    <value>Starts a file watcher and runs a command when files change.</value>
```

Comment:
Keeping with some of the verb tense of the other descriptions, I'd suggest:

`Start a file watch that runs a command when files change.`

Reply:
Changed as suggested